dns/unboundbl/src/opnsense/service/conf/actions.d/actions_unboundbl.conf

```diff
@@ -1,5 +1,5 @@
 [refresh]
-command:/usr/local/opnsense/scripts/OPNsense/Unboundbl/UnboundBL.sh
+command:sh /usr/local/opnsense/scripts/OPNsense/Unboundbl/UnboundBL.sh
```

Comment:
this is not needed, you probably have not set up the executable bit in the file permission correctly.